### ISSUE TITLE
Card Mod only applying style to the first button of manual-alarm-panel after disarm action

### ISSUE THREAD

**Issue by jwestyp** [thomasloven/lovelace-card-mod#68]
I am experiencing a strange error. I am using card-mod for the manual-alarm-panel. When I first load the panel, the new css is applied to all 3 arm buttons, and if I refresh the dashboard it applies again. 
![image](https://user-images.githubusercontent.com/1631496/89655415-b19cfd00-d88f-11ea-9f7d-930a6a5b4202.png)

However when I disarm, the css only applies to the first button.
![image](https://user-images.githubusercontent.com/1631496/89655438-ba8dce80-d88f-11ea-81dd-98ad13216b82.png)

I checked the styles applied and can confirm that the position and font-size are inserted in the computed style only on the first button. I only noticed it this morning as I was disarming the alarm. If I refresh, all is good, but using Fully Kiosk as my tablet browser is a bit of a pain to do on 3 devices every disarm.

**Comment by jcooper-korg** [comment]
I'm seeing this same behavior.  I use card mod to increase the size of the buttons, and show them in red if any doors/windows are open:
```
   cards:
      - entity: alarm_control_panel.house
        name: ' '
        states:
          - arm_home
          - arm_away
        style:
          '#armActions mwc-button':
            $: |
              button {
                font-size: 22px !important;
                --mdc-theme-primary:  {% if is_state('group.all_doors_and_windows', 'off') %} rgb(30, 215, 96); {% else %} red {% endif %};
              }
```
Before arming, it looks right:
<img width="809" alt="Screen Shot 2020-09-21 at 9 48 50 AM" src="https://user-images.githubusercontent.com/1987267/93796679-07cfc080-fbf0-11ea-917b-98ff11028a5f.png">

But after arming, the second button looks as it would without the card mod style adjustments:
<img width="803" alt="Screen Shot 2020-09-21 at 9 48 36 AM" src="https://user-images.githubusercontent.com/1987267/93796618-f5558700-fbef-11ea-9264-e00784b07d74.png">

The web inspector shows the same before/after arming, so not clear why it's rendering differently:
<img width="694" alt="Screen Shot 2020-09-21 at 9 53 26 AM" src="https://user-images.githubusercontent.com/1987267/93796923-64cb7680-fbf0-11ea-9100-9b27d5b1e21e.png">

Any ideas?

**Comment by erock8303** [comment]
This is probably low on the priority list, but just seeing if there has been any movement on this since August.

**Comment by smeehrrr** [comment]
Also hitting this, would love a fix.  Everything is working perfectly for me otherwise.

**Comment by thomasloven** [comment]
This should be fixed in version 3.0.0
